### ISSUE TITLE
Add term wide distal femoral metaphysis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10414]
**Preferred term label:**
Wide distal femoral metaphysis

**Synonyms**
Wide metaphysis of innermost thighbone, broad distal femoral metaphysis

**Definition (free text, please give PubMed ID)**
Increased width of the distal part of the shaft (metaphysis) of the femur.
PMID: 6930380

**Parent term (use hpo.jax.org/app)**
Broad femoral metaphyses HP:0006417

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: PMID: 6930380, PMID: 27577507
Pyle disease (omim265900)
Frontometaphyseal dysplasia (omim305620)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
This was
Wide distal femoral metaphysis HP:0006387
It was missing a parentage, I will fix it.

**Comment by pnrobinson** [comment]
fixed
